docs/DeveloperGuide.adoc

```diff
@@ -767,6 +826,110 @@ Use case ends.
 +
 Use case resumes at step 2.
 
+=== Use case: Sort list
+
+*MSS*
+
+1.  User requests to list persons
+2.  AddressBook shows a list of persons
+3.  User requests to sort list
+4.  AddressBook sorts the list of persons
+5.  AddressBook shows sorted list of persons
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 2a. The list is empty.
++
+Use case ends.
+
+[none]
+* 5a. The list is empty.
++
+Use case ends.
+
+=== Use case: Find person
+
+*MSS*
+
+1.  User requests to find person with keyword (name or first few characters of name)
+2.  AddressBook shows a list of persons with keyword
++
+Use case ends.
+
+*Extensions*
+
+[none]
+
+=== Use case: Add person
+
+*MSS*
+
+1.  User requests to add person
+2.  User types in details of person to add
```

Comment:
use cases should capture the interaction between user and the system. this use case doesn't follow that format